root/static/scripts/common/i18n/expand2.js

```diff
@@ -206,7 +206,7 @@ export const createTextContentParser = <T, V>(
   return mapValue(text);
 };
 
-const varSubst = /^\{([0-9A-z_]+)\}/;
+const varSubst = /^\{([0-9A-z_ ]+)\}/;
```

Comment:
Beginning/ending spaces should probably not be allowed.

Reply:
Is there a reason why they shouldn't? I mean, I don't think we're going to *use* them, but what's the damage?

Reply:
To prevent using them indeed. Damage is broken interpolation and translation. Same for underscores.

Reply:
But that would only be a problem if we actually did add spaces at the beginning or end of variables, or a translator messed up (which is already the case even without the spaces) :) That said, if the regex can be easily changed to your preference, I have nothing against committing a suggestion for it, I just don't think we should complicate it too much otherwise.

Reply:
Actually I don’t think that allowing spaces in variable names is a good approach at all, see [below](https://github.com/metabrainz/musicbrainz-server/pull/3186#issuecomment-1979380835).